packages/manager/src/features/CloudPulse/Alerts/AlertsDetail/AlertDetailOverview.tsx

```diff
@@ -0,0 +1,87 @@
+import { CircleProgress } from '@linode/ui';
+import { Grid, Typography, useTheme } from '@mui/material';
+import React from 'react';
+
+import { useCloudPulseServiceTypes } from 'src/queries/cloudpulse/services';
+
+import { convertStringToCamelCasesWithSpaces } from '../../Utils/utils';
+import { severityMap } from '../constants';
+import { formatTimestamp, getServiceTypeLabel } from '../Utils/utils';
+import { AlertDetailRow } from './AlertDetailRow';
+
+import type { Alert } from '@linode/api-v4';
+
+interface OverviewProps {
+  /*
+   * The alert for which the criteria is displayed
+   */
+  alert: Alert;
+}
+export const AlertDetailOverview = (props: OverviewProps) => {
+  const { alert } = props;
+
+  const {
+    created_by: createdBy,
+    description,
+    label,
+    service_type: serviceType,
+    severity,
+    status,
+    type,
+    updated,
+  } = alert;
+
+  const { data: serviceTypes, isFetching } = useCloudPulseServiceTypes(true);
```

Comment:
Do we need error handling for useCloudPulseServiceTypes? or we will take  it separately as you started working on it now?

Reply:
@kmuddapo , if service types fails, we will not show any error state, we will continue to show details like dbaas, linode etc.,